packages/bierzo-wallet/src/logic/transactions/types/BwSendTransaction/ui/SendTxRow/index.tsx

```diff
@@ -34,11 +34,16 @@ interface Props {
 function SendTxRow({ sendTx, userAddresses }: Props): JSX.Element {
   const classes = useStyles();
   const theme = useTheme<Theme>();
+  const amountClass = sendTx.outgoing ? undefined : classes.amountFrom;
+
   const [isOpen, toggle] = useOpen();
 
   const address = sendTx.outgoing ? sendTx.original.recipient : sendTx.original.sender;
   const amountSign = sendTx.outgoing ? "-" : "+";
-  const amountClass = sendTx.outgoing ? undefined : classes.amountFrom;
+
+  const amount = sendTx.original.amount;
+  const fee = sendTx.outgoing && sendTx.original.fee && sendTx.original.fee.tokens;
```

Comment:
You can even go fancies with [TypeScript 3.7 optional chaining:](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining)

```ts
const fee = sendTx.outgoing && sendTx.original.fee?.tokens;
```

Reply:
I had tried that but go this issue:
![image](https://user-images.githubusercontent.com/44572727/71246960-0c411300-2318-11ea-822b-02cbf811215e.png)

Reply:
1. Check your IDE is using TS 3.7 
<img width="939" alt="Bildschirmfoto 2019-12-20 um 11 04 50" src="https://user-images.githubusercontent.com/2603011/71247295-bcaf1700-2318-11ea-8396-aee4bd356e74.png">
 
2. This is a prettier error. Prettier got updated to support TS 3.7 in https://github.com/iov-one/ponferrada/pull/937. Did you pull current master and base on that? Did you `yarn install`? Did you reload VS Code to restart the eslint plugin?